package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-dotplot",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
Please increase version of the visual to 1.0.1 due to minimal version at Office Store